cmd/podman/buildfarm/create.go

```diff
@@ -0,0 +1,85 @@
+package buildfarm
+
+import (
+	"fmt"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/utils"
+	"github.com/containers/podman/v4/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmCreateDescription = `Create a new farm with connections added via podman system connection add`
```

Comment:
We should probably note that `system connection add --farm` exists.

Reply:
Fixed